doc/classes/@GlobalScope.xml

```diff
@@ -2882,20 +2883,29 @@
 			[b]Note:[/b] The trailing colon is required for properly detecting built-in types.
 		</constant>
 		<constant name="PROPERTY_HINT_NODE_PATH_TO_EDITED_NODE" value="24" enum="PropertyHint">
+			Hints that a [NodePath] property should be relative to the scene root. The inspector editor will be initialized with [constant PROPERTY_USAGE_NODE_PATH_FROM_SCENE_ROOT].
 		</constant>
 		<constant name="PROPERTY_HINT_OBJECT_TOO_BIG" value="25" enum="PropertyHint">
+			Hints that an object is too big to be sent via the debugger.
 		</constant>
 		<constant name="PROPERTY_HINT_NODE_PATH_VALID_TYPES" value="26" enum="PropertyHint">
+			Hints that the hint string specifies valid node types for property of type [NodePath].
 		</constant>
 		<constant name="PROPERTY_HINT_SAVE_FILE" value="27" enum="PropertyHint">
+			Hints that a [String] property is a path to a file. Editing it will show a file dialog for picking the path for file to be saved, with access to the project's directory. The hint string can be a set of filters with wildcards like [code]"*.png,*.jpg"[/code].
```

Comment:
```suggestion
			Hints that a [String] property is a path to a file. Editing it will show a file dialog for picking the path for the file to be saved at. The dialog has the access to the project's directory. The hint string can be a set of filters with wildcards like [code]"*.png,*.jpg"[/code]. See also [member FileDialog.filters].
```